pom.xml

```diff
@@ -159,11 +158,11 @@
   <distributionManagement>
     <snapshotRepository>
       <id>ossrh</id>
-      <url>https://google.oss.sonatype.org/content/repositories/snapshots</url>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
```

Comment:
Is this intentional?

Reply:
Ohh good catch. I think I messed up merging. I will attempt this PR again.